ampcrowd/docker-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Migrate databases if necessary
+sleep 5
+python ampcrowd/manage.py syncdb --noinput
+
+# Process options
+
+# Enable SSL
+if [ "$1" == "-s" ] || [ "$2" == "-s" ]
+then
+    export SSL=1
+else
+    export SSL=0
+fi
+
+python ampcrowd/manage.py collectstatic --noinput
+
+# Print errors directly to console for easy debugging with -d
+if [ "$1" == "-d" ] || [ "$2" == "-d" ]
+then
+    export DEVELOP=1
+    python ampcrowd/manage.py celeryd -l DEBUG &
+    python ampcrowd/manage.py runserver 0.0.0.0:8000
```

Comment:
Didn't realize that... we could do some detection to see if both flags exist, if they do then default to gunicorn, if only `-d` is set, then use runserver.